.expeditor/verify.pipeline.yml

```diff
@@ -34,6 +34,7 @@ steps:
   expeditor:
     executor:
       docker:
+        image: "chefes/lita-worker"
```

Comment:
We should never use this image in Buildkite. We should always use `chefes/buildkite`.